ui/app/src/Views.elm

```diff
@@ -34,8 +34,9 @@ currentView model =
         AlertsRoute filter ->
             AlertList.view model.alertList filter
 
-        SilenceListRoute route ->
-            SilenceList.view model.silences model.silence model.currentTime model.filter
+        SilenceListRoute filter ->
+            -- Is this filter needed???
```

Comment:
@stuartnelson3 @w0rm As far as I can see we can remove both the comment and the `filter` payload from the `SilenceListRoute`.

In addition as the `filter` in the main model does not only hold the filter text but also all other kind of url parameters (e.g. `showSilenced`), what do you think of renaming it to `urlParameters`?